devito/parameters.py

```diff
@@ -236,11 +236,15 @@ class switchconfig(object):
     """
 
     def __init__(self, **params):
```

Comment:
so what's the error trace if `condition=None` were part of the signature instead? Apologies if I had already asked ...

Reply:
https://devitocodes.slack.com/archives/C02LA241E57/p1683717242731189?thread_ts=1683714842.075059&cid=C02LA241E57

Reply:
I've added comments on Slack at the thread above